examples/relay-hook-example/todo/package.json

```diff
@@ -17,7 +17,7 @@
     "react": "^16.8.4",
     "react-dom": "^16.8.4",
     "react-relay": "^4.0.0",
-    "relay-hooks": "1.2.0",
+    "relay-hooks": "file:../../../relay-hooks-1.2.1-a12.tgz",
```

Comment:
fix this?

Reply:
you can also force a resolution to the parent in the webpack config: https://github.com/4Catalyzer/astroturf/blob/2a7bbe84534ed4bd9ac2d1626ce4494b3dceba84/example/webpack.config.js#L45

Reply:
thanks for the suggestion, next time I try to use the method you recommended :)

when I distributed the library I fixed it on the master